### PR TITLE
Bugfix/rm 9414 ios 15 safari bei web controls wird eine falsche textfarbe genutzt

### PR DESCRIPTION
**Pull request by Frazerus**
https://re-motion.atlassian.net/browse/RM-9414

### CODE REVIEW

File: Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css [hunk 1]
```diff
@@ -150,6 +150,7 @@ select.remotion-themed,
 .remotion-themed textarea,
 .remotion-themed select
 {
+  color: var(--color-standard-text); /* not sure about this one */
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css]:
```suggestion
  color: var(--color-standard-text);
```

File: Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css [hunk 2]
```diff
@@ -162,6 +163,7 @@ input[type=button].remotion-themed,
 .remotion-themed input[type=submit],
 .remotion-themed input[type=button]
 {
+  color: var(--color-standard-text); /* not sure about this one */
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css]:
```suggestion
  color: var(--color-standard-text);
```

File: Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css [hunk 3]
```diff
@@ -162,6 +163,7 @@ input[type=button].remotion-themed,
...
   background-color: var(--remotion-themed-background-color-control);
   border: var(--remotion-themed-border-width) solid var(--color-primary);
   border-radius: var(--remotion-themed-border-radius);
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css:169]: Line 189 etc: we should consider also setting the text-color for select > option, too. am worried about Apple-people ;)

File: Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css [hunk 4]
```diff
@@ -18,6 +18,7 @@ span.DropDownMenuSelect
   height: var(--remotion-themed-singleline-min-height);
   line-height: var(--remotion-themed-line-height);
   width: 100%;
+  color: var(--color-standard-text); /*maybe overkill as we set the color on the label below and the other child colors as well */
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css]:
```suggestion
```

File: Remotion/Web/Core/Res/Themes/NovaViso/HTML/SingleView.css [hunk 5]
```diff
@@ -34,6 +34,8 @@ div.singleViewView
   display: flex;
   flex-direction: column;
   overflow: hidden;
+  color: var(--color-standard-text);
+  background-color: var(--color-standard-background);
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaViso/HTML/SingleView.css]:
```suggestion
```
It's just an aera where we place our content. we don't want do do colors here.

File: Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css [hunk 6]
```diff
@@ -13,6 +13,8 @@ input.webButton
   padding: 0;
   box-sizing: border-box;
   display: inline-block;
+  background-color: white;
+  color: black;
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css]: css properties please

**Review by MichaelKetting — CHANGES_REQUESTED** [review]
didn't get to novagray yet.

**Comment by MichaelKetting** [comment]
@Frazerus I realized, there's a couple spots that should probably also get color in BocList, TabbedMenu, TabbedMultiView, SingleView (you already added color there), FormGrid. I'm not 100% sure about the best setup yet. 
You did suggest we should just do body and since we already have font and font-size set on the body, perhaps. 

Right now, e.g. in BocList, we got some parts where the background color and specific foreground accents are not in sync so this sort of mixing I want to cleanup.

In the next screenshot, the red background and the bright green text color bleeds through from the body style. The dark green background and dark red text are based on the standard colors we assign. 

image

**Comment by Frazerus** [comment]
Fixed some things:
Green/red come from body.

image

**Comment by MichaelKetting** [comment]
@Frazerus  thanks, looks good. The sub-menuitem line in the tabbedmenu should also get a background and font color
image

And for the review, I need to verify all parts within BocList are using the correct font and background colors, since the FormGrid is sitting right behind it and could be bleeding through.

**Comment by Frazerus** [comment]
Here is a screenshot of the formgrid having set background to purple and text to coral:
image

Nothing bleeds through :)

**Comment by MichaelKetting** [comment]
Lovely, thanks!

**Comment by Frazerus** [comment]
image

File: Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css [hunk 7]
```diff
@@ -18,15 +18,12 @@ table.tabbedMenu
...
   border-top: var(--tabbedMenu-separator);
   border-bottom: var(--tabbedMenu-separator);
 }
 
-td.tabbedMainMenuCell
-{
-  background-color: var(--color-contrast);
-}
-
 div.tabbedMainMenu div.tabStripTabsPane ul li,
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css]: Just a quick review remark based on code:
color-contrast is gray and was only applied on the mainmenu-row, not the submenu-row. I think this changed the coloring?

Comment (Frazerus) [reply Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css]: Ah, I didn't notice that. Tahnks!

**Comment by Frazerus** [comment]
Fixed branch names so a new PR is required: #1284

**Pull request closed without merging**